lib/pdk/module/build.rb

```diff
@@ -152,6 +158,58 @@ def warn_symlink(path)
         }
       end
 
+      # Checks if the path length will fit into the POSIX.1-1998 (ustar) tar
+      # header format.
+      #
+      # POSIX.1-2001 (which allows paths of infinite length) was adopted by GNU
+      # tar in 2004 and is supported by minitar 0.7 and above. Unfortunately
+      # much of the Puppet ecosystem still uses minitar 0.6.1.
+      #
+      # POSIX.1-2001 tar format does not allow for paths greater than 256 bytes,
```

Comment:
Should this comment refer to the older `POSIX.1-1998` format?

Reply:
Yes, yes it should